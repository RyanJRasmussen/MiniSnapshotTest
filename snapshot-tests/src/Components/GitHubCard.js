import React from "react";
import Card from 'react-bootstrap/Card';
import 'bootstrap/dist/css/bootstrap.min.css';
import Headshot from "../images/Headshot.jpeg"

function GitHubCard() {
    return (
        <Card style={{ width: '18rem' }}>
        <Card.Img variant="top" src={Headshot} />
        <Card.Body>
            <Card.Title>Github Username: RyanJRasmussen</Card.Title>
            <Card.Text>
                Gamer, musician, software dev
            </Card.Text>

        </Card.Body>
        </Card>
    )
}

export default GitHubCard;