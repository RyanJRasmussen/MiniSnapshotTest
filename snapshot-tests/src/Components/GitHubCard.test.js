import GitHubCard from "./GitHubCard"
import renderer from 'react-test-renderer'


test('renders github information card', () => {
    const tree = renderer.create(<GitHubCard/>).toJSON()
    expect(tree).toMatchSnapshot()
})
