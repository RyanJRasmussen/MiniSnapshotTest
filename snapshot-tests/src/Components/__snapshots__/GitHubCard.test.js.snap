// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders github information card 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="Headshot.jpeg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Github Username: RyanJRasmussen
    </div>
    <p
      className="card-text"
    >
      Gamer, musician, software dev
    </p>
  </div>
</div>
`;
